vms/platformvm/camino_service.go

```diff
@@ -475,16 +475,20 @@ func (s *CaminoService) RegisterNode(_ *http.Request, args *RegisterNodeArgs, re
 	return nil
 }
 
+type ClaimedAmount struct {
```

Comment:
What do you think about removing from service? Avax is deprecating all its tx api functions

Reply:
Its already here, so I'd keep i tuntil it will be to hard to support OR until we'll develop convenient way to test txs locally without such API